src/core/qgis.h

```diff
@@ -1250,6 +1250,22 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( AngularDirection )
 
+    /**
+     *  Usage of the renderer.
+     *
+     *  \note This usage not alter how the map gets rendered but the intention is that data provider
```

Comment:
Can you move this note to the QgsRenderContext/QgsMapSettings members instead? It's not a generic property of the enum itself, rather how it's interpreted in the rendering.